### PR TITLE
Make adding curlies more accurate

### PR DESCRIPTION
**Pull request by scottming**
image

Above is the current behavior, which is annoying.

### CODE REVIEW

File: apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs [hunk 1]
```diff
@@ -37,7 +37,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.StructTest do
       assert completion.kind == :struct
     end
 
-    test "should complete module aliases after %", %{project: project} do
+    test "should complete module aliases and ends with {} after %", %{project: project} do
```
Comment (scohen) [inline apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs:40]: what about a test for not adding curlies when it already ends with {} ?

File: apps/server/lib/lexical/server/code_intelligence/completion/translations/module_or_behaviour.ex [hunk 2]
```diff
@@ -18,7 +18,10 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.ModuleOrBehavi
     detail = builder.fallback(module.summary, module.name)
     struct_reference? = Env.struct_reference?(env)
     defines_struct? = Intelligence.defines_struct?(env.project, module.full_name)
-    add_curlies? = defines_struct? and not String.contains?(env.suffix, "{")
+
+    add_curlies? =
```
Comment (scohen) [inline apps/server/lib/lexical/server/code_intelligence/completion/translations/module_or_behaviour.ex:22]: I suggest changing line 47 to `if add_curlies? do`

File: apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs (hunk 1 above)
Comment (scottming) [reply apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs:40]: o, there is already a test to do that; I need to reset this description.

File: apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs [hunk 3]
```diff
@@ -51,6 +51,20 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.StructTest do
...
+        defmodule TestModule do
+        alias Project.Structs.User
+
+        Us|
+      ]
+
+      assert [completion] = complete(project, source)
+
+      assert completion.insert_text == "User"
+      assert completion.kind == :struct
```
[the remark this answers is not in the document]
Comment (scottming) [reply apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs]: right.

**Review by scohen — APPROVED** [review]

**Pull request merged**